.pre-commit-config.yaml

```diff
@@ -14,21 +14,10 @@ repos:
       pass_filenames: false
       require_serial: true
       args: ["."]
-  - repo: local
+  - repo: https://github.com/psf/black
```

Comment:
Could the black changes not go in a separate PR?

Reply:
Created. #1763